stacker_blueprints/iam_roles.py

```diff
@@ -92,17 +147,24 @@ def create_policy(self, name):
         )
 
         t.add_output(
-            Output(name + "PolicyName", Value=Ref(policy))
+            Output("PolicyName", Value=Ref(policy))
```

Comment:
This could be a breaking change for people that use this blueprint already, right?  If this output was relied on in another stack, that stack will fail to update when this changes.  Not sure that's a huge deal - any opinion here @aarcro (since you were the original author, so I assume you might be using this blueprint already) @ejholmes @Lowercases @russellballestrini  does remind use this?